src/main/java/randoop/main/Minimize.java

```diff
@@ -1090,15 +1130,11 @@ public String call() throws Exception {
    * @param compUnit the compilation unit to write to file
    * @param file file to write to
    */
-  private static void writeToFile(CompilationUnit compUnit, File file) {
+  private static void writeToFile(CompilationUnit compUnit, File file) throws IOException {
     // Write the compilation unit to the file.
-    try (BufferedWriter bw = Files.newBufferedWriter(file.toPath(), UTF_8)) {
```

Comment:
What is the reason for undoing the try-with-resources statement?  If you use try-with-resources, then the call to `close()` is not necessary, and the BufferedWriter is closed even if an exception is thrown.